manager.go

```diff
@@ -422,6 +423,24 @@ func (m *Manager) dial(ctx context.Context) error {
 	return nil
 }
 
+// process non-subsystem effects of a config (i.e. agent-specific stuff that needs to happen when loading cache and when updating).
+func (m *Manager) processConfig(cfg map[string]*pb.DeviceSubsystemConfig) {
```

Comment:
I feel like this doesn't really belong here, but in viamagent/viamagent.go file, as the Update() and NewSubsystem() functions there are specifically meant to handle config changes. (Though I get there's the subtle problem of how early in startup can this be set.)

I suppose this is fine for now though. There's a scope coming down the pipe likely to reorganize the config structure soon, so we can consolidate configuration handling better after/during that I think. Viam-server's "fast_start" and "start_timeout" are other special cases that are handled entirely uniquely too right now.

Reply:
yeah I modeled after fast_start -- agree with changing (but later)